CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+
+### Discuss your design
+
+All contributions are welcome, but please let everyone know what you are working on. The way to do this is to first file an issue (or claim an existing issue). In this issue, please, discuss what your plan is to fix or add the feature. Also, all design discussions should happen on the issue. If design discussions happen in a channel, reconcile the decisions to the relevant issue(s). Once, your contribution is ready, create a pull request referencing the issue. Once, a pull request is created one or more of the Core Contributors will review the pull request and may request changes. Once the changes are approved, it will be merged into the current release candidate branch.
+
+Be sure to keep the pull request updated as merge conflicts may occur as other things get merged into the release branch before yours.
+
+Please, note that we may push your pull request to the next release candidate at which point you will have to resolve any conflicts that occur.
+
+### Not sure where to contribute?
+
+Want to contribute but not sure where? Not a problem, the best thing to do is look through the issues and find one that interests you. If the issue has the label `good first issue`, it means that one of the core contributors thinks this is a good issue to start with. But, this doesn’t mean that you have to start with these issues. Go through the issues and see if someone is already working on it. If no one is, state that you will be working on the issue to claim it. If you are unsure where to start on the issue, ask in the issue and one of the Core Contributors will help you out.
+
+## How to build from source
+
+If you just need to play with the binary the easy way is to get the binary from our [releases page](https://github.com/go-spatial/tegola/releases), or if a binary for your OS is not there, use `go get -u github.com/go-spatial/tegola/cmd/tegola`.
+
+If however you want to build the latest release candidate you will have to build from source. The first thing to do is to clone the `go-spatial/tegola` repo to your `GOPATH`. The simplest way to do this is to use `go get -u github.com/go-spatial/tegola`, navigate to the repository root then: 
+
+* Checkout the current release candidate branch, (i.e. v0.7.0)
+	
+	(`git checkout v0.7.0`)
+	
+* Create a new feature branch. 
+	
+	(`git checkout -b issue-XXX-new_feature`)
+	
+* Work on the fix, and run all the tests. We need to run tests with CGO enabled and disabled.
+	
+	(`go test ./…`)
+ 	(`CGO_ENABLED=0 go test ./…`)
+	
+* Commit your changes (`git commit -am ‘Add some feature #XXX\n\nExtened description.'`)
+
```

Comment:
The second paragraph and list explain how to add a new branch but not how to build the release candidate from source